pkg/controller/instanceset/pod_role_event_handler_test.go

```diff
@@ -52,7 +52,7 @@ var _ = Describe("pod role label event handler test", func() {
 				Namespace:  pod.Namespace,
 				Name:       pod.Name,
 				UID:        pod.UID,
-				FieldPath:  readinessProbeEventFieldPath,
+				FieldPath:  lorryEventFieldPath,
```

Comment:
lagacyReadinessProbeEventFieldPath?

Reply:
This test tests PodRoleEventHandler's behaviour. IMO It doesn't matter which fieldpath to use.